### PR TITLE
add demo link to README

### PR DESCRIPTION
**Pull request by dehall**
also added a TM and removed the word prototype from the title because it spilled over onto the next line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,11 @@
 [![Powered by Synthea](https://img.shields.io/badge/powered%20by-synthea-informational.svg)](https://github.com/synthetichealth/synthea)
 [![Apache 2.0 License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.txt)
 
-# Camino: mCODE™-based Oncology Clinical Pathways Prototype
+# Camino™: mCODE™-based Oncology Clinical Pathways
 Camino is a prototype SMART on FHIR application intended to demonstrate the value of mCODE, a standardized data model around cancer, through computable oncology clinical pathways. Camino uses mCODE data in the patient's record fetched from the EHR to automatically evaluate the patient's current location on a structured pathway and provides recommendations as to next steps.
 
+# [Live Demo](https://mcode.github.io/pathways/#/demo)
```
Comment (radamson) [inline README.md]: Should this be a a level 2 header?  I'm cool with it either way regardless.

```suggestion
## [Live Demo](https://mcode.github.io/pathways/#/demo)
```

**Review by radamson — APPROVED** [review]

**Comment by blangley28** [comment]
It would be nice if the link opened in a new tab. I've never done it in markdown but it looks like it might be possible: https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank
If it doesn't work no big deal

**Comment by dehall** [comment]
Per that link it doesn't seem like it's possible on GitHub

**Comment by blangley28** [comment]
Guess I should've kept reading passed the first comment!

**Review by blangley28 — APPROVED** [review]

**Pull request merged**